x/protocolpool/keeper/keeper.go

```diff
@@ -188,13 +199,55 @@ func (k Keeper) SetToDistribute(ctx context.Context, amount sdk.Coins, addr stri
 		return err
 	}
 
+	totalStreamFundsPercentage := math.ZeroInt()
+	err = k.RecipientFundPercentage.Walk(ctx, nil, func(key sdk.AccAddress, value math.Int) (stop bool, err error) {
+		totalStreamFundsPercentage = totalStreamFundsPercentage.Add(value)
+		return false, nil
+	})
+	if totalStreamFundsPercentage.GT(math.NewInt(100)) {
+		return fmt.Errorf("total funds percentage cannot exceed 100")
```

Comment:
Is there a check anywhere to ensure the total never exceed 100%?

This gets called in distribution endblocker which would halt the chain of governance allocated too much streams.

Reply:
oh, you're right!
I didn't have a check for this.
I added a check before creating a continuous fund proposal to find if the total percentage then exceeds 100%. If yes, then setting up a new proposal fails.